code/modules/mob/living/living.dm

```diff
@@ -570,7 +570,8 @@ Sorry Giacom. Please don't be mad :(
 	else
 		stop_pulling()
 		. = ..()
-	if ((s_active && !( s_active in contents ) ))
+	if (s_active && !(s_active in contents) && !(s_active.loc in contents))
+		// It's ugly. But everything related to inventory/storage is. -- c0

```

Comment:
Didnt spot that one in the diff. My shits a little bit outdated. 
Also can someone please tell me if the action buttons thing is right or not so I can fix it